app/models/quiz_question_answer.rb

```diff
@@ -28,4 +28,5 @@ class QuizQuestionAnswer < ApplicationRecord
   has_many(:answering_participations, through: :selections, source: :participation)
 
   scope :correct, -> { where(is_correct: true) }
+  scope :incorrect, -> { where(is_correct: false) }
```

Comment:
I used to pretty much try to only add application code that is used by other application code. This `incorrect` scope is only used in a test, though. But I guess I am feeling a little more okay with that, now. I think that we can say that the purpose of application code is not solely to serve the end users. I think it's okay for at least some of it to be helpful to developers.